src/components/AuthorViewpoint/EditViewpointFact.tsx

```diff
@@ -0,0 +1,48 @@
+import { Fact } from "@/types/conversations.types";
+import { Button } from "@mantine/core";
+import { XMarkIcon } from "@heroicons/react/24/outline";
+import Link from "next/link";
+
+type FactCardProps = {
+    fact: Fact;
+    removeFact: (id: string) => void;
+};
+
+export default function EditViewpointFact({ fact, removeFact }: FactCardProps) {
+    return (
+        <div>
+            <div className="group flex">
+                <h1 className="float-left text-lg font-normal text-black">
+                    {fact.title}
+                </h1>
+                <Button
+                    variant="transparent"
+                    classNames={{
+                        root: "float-right pr-1 pl-0 flex opacity-0 group-hover:opacity-100",
```

Comment:
Consider using the `hidden` class instead of setting opacity

Reply:
To prevent layout shifts when toggling the visibility of an element, instead of using display to show or hide it—which removes the element from the layout entirely—I use the visibility property. This approach visually hides the element while keeping its space reserved in the layout, ensuring a stable and consistent page structure.

Reply:
Understood, in this case it does make more sense to use opacity. Nice work!